server/lib/auth0.js

```diff
@@ -88,7 +127,7 @@ export const deleteRules = (progress, client, rules) => {
     return Promise.resolve(true);
   }
 
-  progress.log('Deleting rules that no longer exist in the repository...');
+  progress.log('Deleting all rules to replace them with the ones existing in the repository...');
```

Comment:
Deleting all rules for every deployment isn't an option. High traffic customers will be impacted by this, because fora brief moment users will be logging in without any rules being applied (which might weaken authorization logic, cause tokens to not contain the required info, ...).

Is this to solve the rules order issue? I think we just need to provide guidance here. Instead of having rules that follow each other directly (like order 1, 2, 3, 4, 5) we should just leave enough space between rules to allow moving them around without issues (eg: order 10, 20, 30, 40, 50)